Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -805,6 +813,31 @@ extension RestRequest: URLSessionDelegate {
         let warning = "Attempting to establish a secure connection; This is only supported by macOS 10.6 or higher. Resorting to default handling."
 
         switch (method, host) {
+        case (NSURLAuthenticationMethodClientCertificate, baseHost):
+            #if !os(Linux)
+            if let certificatePath = self.certificatePath {
+                // Get the bundle path from the Certificates directory for a certificate that matches clientCertificateName's name
+                if let bundle = Bundle.path(forResource: self.clientCertificateName, ofType: "der", inDirectory: certificatePath) {
+                    // Convert the bundle path to NSData
+                    if let key: NSData = NSData(base64Encoded: bundle, options: .ignoreUnknownCharacters) {
+                        // Create a secure certificate from the NSData
+                        if let certificate: SecCertificate = SecCertificateCreateWithData(kCFAllocatorDefault, key) {
+                            // Create a secure identity from the certificate
+                            var identity: SecIdentity? = nil
+                            let _: OSStatus = SecIdentityCreateWithCertificate(nil, certificate, &identity)
```

Comment:
https://developer.apple.com/documentation/security/1401160-secidentitycreatewithcertificate says "Call the CFRelease function to release this object when you are finished with it." so I guess we need to do that?

Reply:
When attempting to use it, it returns the following error: 'CFRelease is unavailable: Core Foundation objects are automatically memory managed', suggesting that this is handled by itself.

Reply:
OK cool.  Bad docs!